src/modules/toolbox/components/shareButton/ShareDialogContent.js

```diff
@@ -5,20 +5,32 @@ import shareIcon from './assets/share.svg';
 import css from './shareDialogContent.css';
 import { emitNotification, LevelTypes } from '../../../../store/notifications/notifications.action';
 import { MvuElement } from '../../../MvuElement';
+import { isHttpUrl } from '../../../../utils/checks';
 
 const Switch_Toggle = 'switch_toggle';
+const Update_Clear = 'update_clear';
+const Update_Url = 'update_url';
+const Update_File_Save_Url = 'update_file_save_url';
 
 /**
- * A content component to show and share perma-links of
- * user-generated measurement- or drawing-data
+ * The published urls of a file (FileSaveResult).
+ * @typedef {Object} FileSaveUrl
+ * @property {string} adminId The url to the adminId version of a file
+ * @property {string} fileId The url to the fileId version of a file
+ */
```

Comment:
Would say documentation is unneeded as already present:
https://github.com/ldbv-by/bav4/blob/d9c17d5da2fa610f610cf2009af4eb7361abac31/src/services/FileStorageService.js#L64